src/views/Log.vue

```diff
@@ -382,7 +382,7 @@ export default {
     }, 500)
 
     /** AutoScroll? */
-    const autoScroll = useInitialOptions('autoScroll', { props, emit }, false)
+    const autoScroll = useInitialOptions('autoScroll', { props, emit }, true)
```

Comment:
Did we discuss this being the default?

Reply:
With the scroll-detection stuff working, I think this should be the default.

Reply:
> Not sure I agree, on the basis that there is no easy way to scroll back to the top if you didn't want it set on. Whereas if you turn it on it will scroll to the bottom for you with one click.

Reply:
Page Up?

Reply:
I don't think a lot of users know they can use home/end keys, or 2-finger scrolling etc!

There are also files where it doesn't make sense to scroll to the bottom, such as the config log and job script

Reply:
Meh, I think autoscroll will still come out favourably on balance.

If you are concerned about people struggling to scroll to the top of the page, it would be easy to add a "scroll to top" button an stick it under the log file.

Reply:
It would be good for us to get https://github.com/cylc/cylc-ui/issues/1146 sorted at some point to allow easier configuration of user preferences.

Reply:
Personally, I'd estimate 75% of the time I open the log view I want to go to the bottom, but it automatically happening the other 25% of the time would outweigh the convenience. I just don't see a strong need to turn it on automatically when it can so easily be turned on with the button or by scrolling to the bottom.